recipe/meta.yaml

```diff
@@ -24,97 +22,98 @@ source:
     folder: src/gpl
 
 {% set lc, vs, ve, prefix, parbuild = "\\", "$", "", "$PREFIX", "-j $CPU_COUNT" %}  # [unix]
-{% set lc, vs, ve, prefix, parbuild = "^", "%", "%", "%LIBRARY_PREFIX%", "" %}      # [win]
 build:
-  number: 1002
+  number: 0
   features:
-    - blas_{{ blas_variant }}  # [not win]
-  skip: true  # [win32 or (win and vc<14)]
+    - blas_{{ blas }}
+  skip: true  # [win32 or win]
   script: |
     mkdir build
     cd build
-
-    cmake ..                                 {{ lc }}
-      -G"{{vs}}CMAKE_GENERATOR{{ve}}"        {{ lc }}
-      -DCMAKE_PREFIX_PATH="{{ prefix }}"     {{ lc }}
-      -DCMAKE_INSTALL_PREFIX="{{ prefix }}"  {{ lc }}
-      -DCMAKE_BUILD_TYPE=Release             {{ lc }}
-      -DBUILD_EXAMPLES=OFF                   {{ lc }}
-      -DBUILD_META_EXAMPLES=OFF              {{ lc }}
-      -DBUNDLE_JSON=OFF                      {{ lc }}
-      -DBUNDLE_NLOPT=OFF                     {{ lc }}
-      -DUSE_SVMLIGHT=OFF                     {{ lc }}
-      -DENABLE_TESTING=OFF                   {{ lc }}
-      -DENABLE_COVERAGE=OFF                  {{ lc }}
-      -DUSE_SWIG_DIRECTORS=ON
-    if errorlevel 1 exit 1  # [win]
-
+    cmake ..                                                  {{ lc }}
+      -G"{{vs}}CMAKE_GENERATOR{{ve}}"                         {{ lc }}
+      -DCMAKE_PREFIX_PATH="{{ prefix }}"                      {{ lc }}
+      -DCMAKE_INSTALL_PREFIX="{{ prefix }}"                   {{ lc }}
+      -DCMAKE_BUILD_TYPE=Release                              {{ lc }}
+      -DBUILD_EXAMPLES=OFF                                    {{ lc }}
+      -DBUILD_META_EXAMPLES=OFF                               {{ lc }}
+      -DBUNDLE_JSON=OFF                                       {{ lc }}
+      -DBUNDLE_NLOPT=OFF                                      {{ lc }}
+      -DUSE_SVMLIGHT=OFF                                      {{ lc }}
+      -DENABLE_TESTING=OFF                                    {{ lc }}
+      -DENABLE_COVERAGE=OFF                                   {{ lc }}
+      -DUSE_SWIG_DIRECTORS=ON                                 {{ lc }}
+      -DLICENSE_GPL_SHOGUN=OFF                                {{ lc }}  # [bla_vendor == 'Intel']
+      -DUSE_SVMLIGHT=OFF                                      {{ lc }}  # [bla_vendor == 'Intel']
+      -DLAPACK_LIBRARIES={{ prefix }}/lib/{{ blas_lib }}      {{ lc }}  # [bla_vendor == 'Intel']
+      -DBLAS_LIBRARIES={{ prefix }}/lib/{{ blas_lib }}        {{ lc }}  # [bla_vendor == 'Intel']
```

Comment:
I don't think you need the above lines for Intel.

Reply:
even the parts about the GPL code? i'm not so sure about the legality things here, but i thought GPL code cannot be linked against MKL

Reply:
I didn't mean Line 46 and 47. I meant only 48 and 49